quantmsrescore/ms2rescore.py

```
@@ -398,136 +269,85 @@ def filter_out_artifact_psms(
-        spectrum_path=spectrum_path,
-        fasta_file=fasta_file,
+        processes=processes,
         id_decoy_pattern=id_decoy_pattern,
         lower_score_is_better=lower_score_is_better,
         log_level=log_level,
         spectrum_id_pattern=spectrum_id_pattern,
-        psm_id_pattern=psm_id_pattern
+        psm_id_pattern=psm_id_pattern,
     )
     logging.info("MS²Rescore config:")
     logging.info(config)
```

Ruff (0.8.2)

Undefined name `config`

(F821)